.eslintrc.cjs

```diff
@@ -3,7 +3,10 @@ module.exports = {
     node: true,
     es2021: true,
   },
-  extends: 'eslint:recommended',
+  extends: [
+    'eslint:recommended',
+    'plugin:node/recommended',
+  ],
```

Comment:
Then comes configuration. The file `.eslintrc.js` (or `.eslintrc.cjs`, when using the `"type": "module"` option in `package.json`) is the default location for eslint configuration. 

The quickest way to get a simple setup here is to `extends: 'eslint;recommended'`, as we had before. This is extending some existing configuration file that is built in to `eslint`, containing all the most generic recommended rules to have.

As you add plugins, they often come with their own set of recommendations, so we add those here as well.